plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -35,12 +35,21 @@
             obtained from
             GET /1.0/containers/<name>
             U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10containersname)
-            are different, they this module tries to apply the configurations.
```

Comment:
It's strange that this sentence is missing. Can you re-add it?

Reply:
I don't have a sentence like that at that position. I also searched for "tries" and found it.

Reply:
i have no idea where the sentence come from. Maybe i have fix them already.

Reply:
In the main branch, this paragraph currently is:
```
          - If the container already exists and its "config" value in metadata
            obtained from
            GET /1.0/containers/<name>
            U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10containersname)
            are different, they this module tries to apply the configurations.
```
(https://github.com/ansible-collections/community.general/blob/main/plugins/modules/cloud/lxd/lxd_container.py#L34-L38)
It's the last line of that paragraph that you removed in this PR.

Reply:
config:
        description:
          - 'The config for the container (e.g. {"limits.cpu": "2"}).
            See U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-1)'
          - If the container already exists and its "config" value in metadata
            obtained from GET /1.0/containers/<name>
            U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10containersname)
          - The key starts with 'volatile.' are ignored for this comparison when I(ignore_volatile_options=true).
          - Not all config values are supported to apply the existing container.
            Maybe you need to delete and recreate a container.
        type: dict
        required: false

Reply:
In my fork the comment is fixed. Maybe i have to make a new PR?

Reply:
It's not really fixed, since that sentence is basically stopping in the middle (after the URL).

Reply:
Please look at: https://github.com/conloos/community.general/blob/main/plugins/modules/cloud/lxd/lxd_container.py there is no broken sentence left. "Maybe i have to make a new PR?"

Reply:
The sentence in https://github.com/conloos/community.general/blob/main/plugins/modules/cloud/lxd/lxd_container.py#L34-L36 is incomplete:
`"If the container already exists and its "config" value in metadata obtained from GET /1.0/containers/<name> U(...url...)"`
1. There is no period at the end.
2. The condition in the "if" part is not complete. The condition consists of two parts: "the container already exists", and then the incomplete part: "its 'config' value in metadata (that is obtainde from GET ...)" - what is it that 'config' value satisfies?
3. There's only a "if" part: what happens if these conditions are satisfied?

Reply:
Can you replace this paragraph by
```
          - If the container already exists and its "config" values in metadata
            obtained from GET /1.0/containers/<name>
            U(https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10containersname)
            are different, this module tries to apply the configurations.
```
? This restores the original final part of the sentence, fixes an error in it (removed superfluous 'they'), and fixes a plural error in the first line ('value' → 'values').

Reply:
Fixed in 6602713b02f563bf1f78de4757c427bd1e9a921e.